ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/title/clientlibs/.content.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0"
+	jcr:primaryType="cq:ClientLibraryFolder"
+    allowProxy="{Boolean}true"
+    categories="[hclecomm.title,hclecomm.content]"/>
```

Comment:
@rakeshkumar-verma : why doe we have 2 categories here ? can go with hclecomm.title

Reply:
We can go but added hclecomm.content for all content component for future perspective. Let me know if want me remove hclecomm.content.

Reply:
@rakeshkumar-verma : please remove . we can refer to same clientlib-  hclecomm.title,

Reply:
@rakeshkumar-verma : have you incorporated this change of removing hclecomm.content ?

Reply:
done